### ISSUE TITLE
buildozer command for reproducibility doesn't work with bzlmod

### ISSUE THREAD

**Issue by alexeagle** [bazel-contrib/rules_oci#315]
A `MODULE.bazel` file like
```
oci.pull(
    name = "distroless_base",
    tag = "latest",
    # digest = "sha256:ccaef5ee2f1850270d453fdf700a5392534f8d1a8ca2acda391fbb6a06b81c86",
    image = "gcr.io/distroless/base",
    platforms = [
        "linux/amd64",
        "linux/arm64",
    ],
)
```

results in 

```
WARNING: for reproducible builds, a digest is recommended.
Either set 'reproducible = False' to silence this warning,
or run the following command to change oci_pull to use a digest:

buildozer 'set digest "sha256:73deaaf6a207c1a33850257ba74e0f196bc418636cada9943a03d7abea980d6d"' 'remove tag' 'remove platforms' 'add platforms "linux/amd64" "linux/arm64" "linux/arm" "linux/s390x" "linux/ppc64le"' WORKSPACE:rules_oci~1.2.0~oci~distroless_base
```

which clearly doesn't work, as it references the WORKSPACE file.

@fmeum recently improved buildozer to support `use_repo` calls - do you have any recommendation if there's an easy way for us to print a command that works for all users?

**Comment by fmeum** [comment]
`//MODULE.bazel:distroless_base` instead of `WORKSPACE:rules_oci~1.2.0~oci~distroless_base` should work (the `//` is optional, but it doesn't require the user to be in the root directory of the project).

**Comment by fmeum** [comment]
Just `MODULE.bazel:distroless_base` gives you better compatibility. The interesting takeaway is that `bazel_dep` and tags with a `name` attribute are recognized as "rules" by buildozer, so all the usual commands just work for them.
